Agent configuration docs: Increase timeout limit for scripts

marko-lisica opened the pull request:
Docs changes related to: #16645

noahtalerman commented on docs/Configuration/agent-configuration.md:
````diff
@@ -441,6 +441,25 @@ agent_options:
+- Config file format:
+  ```yaml
+  agent_options:
+    config:
+      options:
+        distributed_interval: 3
+        distributed_tls_max_attempts: 3
+        logger_tls_period: 10
````
```suggestion
    config:
```

Maybe we cut all the keys that don't correspond w/ script execution timeout? To make the example clearer (less words). It looks like we could make this same change to other examples in the agent options docs.

marko-lisica replied:
I agree. I committed a suggestion.

noahtalerman replied:
Looks good! Can you please make this same change for the other examples on this doc page? I think it should be quick.

marko-lisica commented:
@noahtalerman I just updated all examples. I realized some of them had only

```
agent_options:
  ...
```
and some of them had:

```
apiVersion: v1
kind: config
spec:
  agent_options:
  ...
```

I removed `apiVersion: v1, kind...` from each example and left it only on the top of the doc in "Example Agent options YAML"

noahtalerman commented:
Hey @eashaw, when you get the chance can you please take a look at this PR?

Waiting to close the associated story (#16645) until we merge in reference changes.
